content/applications/hr/payroll/reporting.rst

```diff
@@ -2,68 +2,103 @@
+If a report is unavailable to a user, an :guilabel:`Invalid Operation` pop-up window appears,
+stating: `You must be logged in to a (country) company to use this feature.`
+
+Filters
+=======
+
+At the top of each report, the default filters are shown inside the :guilabel:`Search` box.
 
-At the top of each report, click on :guilabel:`Filters` to display the optional filters for the
-specific report. Filters show information that matches the specific filter parameters. For example,
-when selecting the :guilabel:`Last 365 Day Payslip` filter, only payslips for the last 365 days
-appear. All reports have the option to add a custom filter, and each have different default custom
-filter options. Select the parameters, then click the :guilabel:`Apply` button.
+Click the :guilabel:`drop-down arrow` in the search box to display the available filters. Filters
+show information that matches the specific filter parameters.
+
+.. example::
+   The :guilabel:`Work Entries Analysis` report has two default filters, the `Current month:(Month)
+   (Year)` filter, and the `Validated` filter. The :guilabel:`Payroll` report only has one default
+   filter, the `Last 365 Days Payslip` filter.
 
 .. image:: reporting/custom-filter.png
    :align: center
    :alt: Report dashboard view.
 
-Clicking on :guilabel:`Favorites` displays three options: :guilabel:`Save the current search`,
-:guilabel:`Add the search to the dashboard`, or :guilabel:`Insert the search to a Google
-spreadsheet`.
+All reports have the option to add custom filters, or group the information by different metrics
+(employee, department, company, etc.). Some reports have the option to compare the current report to
+the previous time period or year (a :guilabel:`Comparison` option).
 
-.. image:: reporting/favorites.png
-   :align: center
-   :alt: Favorite options for reports.
+Click on a parameter to select it and activate it. The report is immediately updated with the new
+parameters.
+
+The updated report can be set as a favorite report, meaning the parameters are stored for quick
+access in the future. First, click :guilabel:`Save the current search` under the
+:guilabel:`Favorites` section, and two options and a :guilabel:`Save` button appear.
+
+If the current report is desired to be the default when the report is accessed, tick the box next to
+:guilabel:`Default filter`. If the current report should be accessible to everyone in the database,
+tick the box next to :guilabel:`Share`.
+
+Finally, click :guilabel:`Save`, which saves the currently configured report, and it now appears
+beneath :guilabel:`Favorites`.
+
+Default reports
+===============
 
 Payroll
 -------
 
-Click on :menuselection:`Payroll --> Reporting --> Payroll` to display the :guilabel:`Payroll
-Analysis` report.
+Click on :menuselection:`Payroll app --> Reporting --> Payroll` to display the :guilabel:`Payroll
+Analysis` report. This report shows all the payslips generated in the last 365 days.
```

Comment:
If I resize the screen at all, you lose either the right half or the bottom which tells you what the info is. Since I didn't think the text was SUPER important, I went with the whole chart since you can see what the screen should look like, and where the buttons are.

I honestly don't know how I can take a different/better image while also showing all the data I need/want to.